File: templates/templates/FSharp.EpoxyHello.Avalonia/EpoxyHello.UI/Views/MainWindow.axaml.fs
namespace EpoxyHello.Views

open Avalonia
open Avalonia.Controls
open Avalonia.Markup.Xaml

[<Sealed>]
type MainWindow() as self =
    inherit Window()

    do
        self.InitializeComponent()
#if DEBUG
        self.AttachDevTools()
#endif

    member private self.InitializeComponent() =
        AvaloniaXamlLoader.Load(self)

File: templates/templates/EpoxyHello.Avalonia11/EpoxyHello.UI/Views/MainWindow.axaml.cs
using Avalonia.Controls;
using Avalonia.Markup.Xaml;

namespace EpoxyHello.Views;

public sealed partial class MainWindow : Window
{
    public MainWindow() =>
        InitializeComponent();

    private void InitializeComponent() =>
        AvaloniaXamlLoader.Load(this);
}
